### PR TITLE
[AIRFLOW-649] Support non-sched DAGs in LatestOnlyOp

### PR DESCRIPTION
**Pull request by r39132**
Dear Airflow Maintainers,

Please accept this PR that addresses the following issues:
- https://issues.apache.org/jira/browse/AIRFLOW-649

Testing Done:
- tested locally

Here's a screen shot, which shows 1 scheduled run and 2 manually triggered runs.
https://www.dropbox.com/s/vch7ohdx5he7xnw/Screenshot%202016-12-21%2023.10.22.png?dl=0

### CODE REVIEW

**Comment by r39132** [comment]
@criccomini

File: airflow/operators/latest_only_operator.py [hunk 1]
```diff
@@ -32,6 +32,13 @@ class LatestOnlyOperator(BaseOperator):
     ui_color = '#e9ffdb'  # nyanza
 
     def execute(self, context):
+        # If the DAG Run is externally triggered, then return without
+        # skipping downstream tasks
+        if context['dag_run'].external_trigger:
+            logging.info("""Externally triggered DAG_Run:\n
```
Comment (criccomini) [inline airflow/operators/latest_only_operator.py]: Nit: not sure about newline in log message. Usually this is poor form, as it breaks parsers and whatnot.

**Review by criccomini — APPROVED** [review]
LGTM

Left a nit. If you feel compelled, go ahead and fix. Either way, good with a merge.

**Comment by r39132** [comment]
@criccomini Removed newline!

**Pull request closed without merging**